docs/static/releasenotes.asciidoc

```diff
@@ -24,6 +26,97 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-6-0]]
+=== Logstash 6.6.0 Release Notes
+
+
+=== Logstash Pull Requests with label v6.6.0
```

Comment:
The doc won't build as is.  I'm guessing that asciidoc doesn't like two headings of the same level without text between.  I got it to build when I deleted `=== Logstash Pull Requests with label v6.6.0`

Reply:
Will remove